mantidproject/mslice
# Interactive cuts

Allow the user to specify a region of a slice and display a dynamically updating cut of that area. Includes the ability to resize the area (by dragging the sides) or move it (by clicking inside and dragging it). 

**To test:**
- Plot a slice
- Click 'Interactive cuts' in menu
- Click and drag to draw a line onto the plot
- Cut of area should appear and update as the area is moved or resized

Fixes #230
